admin/class-admin-apple-bulk-export-page.php

```diff
@@ -96,7 +96,27 @@ public function ajax_push_post() {
 		// Sanitize input data
 		$id = absint( $_GET['id'] );
 
-		// TODO: Move push action to shared
+		// Ensure the post exists and that it's published
+		$post = get_post( $id );
+		if ( empty( $post ) ) {
+			echo json_encode( array(
+				'success' => false,
+				'error'   => __( 'This post no longer exists.', 'apple-news' ),
+			) );
+			wp_die();
```

Comment:
It can't because the original developers used `error` and not `data`, which is what `wp_send_json_error` will transmit.